### ISSUE TITLE
[2.6 RC1 - enhancement] Scroll Automation results

### ISSUE THREAD

**Issue by xJoe** [MilosKozak/AndroidAPS#2256]
Sorting Automation rules is great! Would be really helpful if the results displayed on the lower part of the page would be scrollable. Can only see the last 3-4. Thanks

**Comment by Philoul** [comment]
Yes sorting is an improvement, but I prefer a dedicated button to delete a rule as in 2.5.1...
I use swipe left or right to switch between tabs, and when I arrive in automation tab, AAPS wants to delete a rule when i try to go back previous tab :-(
